openedx_events/bridge/avro_attrs_bridge.py

```diff
@@ -1,120 +1,139 @@
 """
-Code to convert attr classes to avro specification.
+Code to convert attr classes to Avro specification.
+
+TODO (EventBus): handle optional parameters and allow for schema evolution (ARCHBOM-2013)
 """
-import io
 import json
-import uuid
-from datetime import datetime
 from typing import Any, Dict
 
 import attr
 import fastavro
 
-from openedx_events.bridge.avro_attrs_bridge_extensions import DatetimeAvroAttrsBridgeExtension
-from openedx_events.bridge.avro_types import AVRO_TYPE_FOR
+from openedx_events.bridge.avro_attrs_bridge_extensions import (
+    CourseKeyAvroAttrsBridgeExtension,
+    DatetimeAvroAttrsBridgeExtension,
+)
+from openedx_events.bridge.avro_types import PYTHON_TYPE_TO_AVRO_MAPPING
 
 
 class AvroAttrsBridge:
```

Comment:
Assuming you decided to update names in a follow-up PR, maybe we could at least acknowledge that decision in a comment here:
```
TODO: Although the bridge implementation has been updated from just an attrs data class bridge, to an OpenEdxPublicSignal event data bridge, the class name and file names will be updated in a follow-up PR. See ARCHBOM-XXXX for details.
```